src/middlewares/polaris-middleware.ts

```diff
@@ -1,23 +1,19 @@
 import { GraphqlLogger } from '@enigmatis/utills';
 import { inject, injectable } from 'inversify';
 import { MiddlewaresConfig } from '../common/injectable-interfaces';
-import { isRepositoryEntity } from '../dal/entities/repository-entity';
-import { MiddlewaresConfiguration } from '../http/request/polaris-request-headers';
 import { POLARIS_TYPES } from '../inversion-of-control/polaris-types';
 import { GraphqlLogProperties } from '../logging/graphql-log-properties';
 import { PolarisContext } from '../server/polaris-context';
+import { FilterExecutor } from './filter-executor';
 import { Middleware, RequestMiddlewareParams, ResponseMiddlewareParams } from './middleware';
-import { DataVersionFilter } from './middleware-activation-condition/filter-data-version';
-import { RealityIdFilter } from './middleware-activation-condition/filter-realities';
-import { SoftDeleteFilter } from './middleware-activation-condition/filter-soft-delete';
 
 @injectable()
 export class PolarisMiddleware implements Middleware {
     @inject(POLARIS_TYPES.GraphqlLogger) polarisLogger!: GraphqlLogger<PolarisContext>;
-    middlewaresConfig: MiddlewaresConfiguration;
+    filterResolver: FilterExecutor;
 
     constructor(@inject(POLARIS_TYPES.MiddlewaresConfig) middlewaresConfig: MiddlewaresConfig) {
-        this.middlewaresConfig = middlewaresConfig.middlewaresConfiguration;
+        this.filterResolver = new FilterExecutor(middlewaresConfig.middlewaresConfiguration);
     }
 
     preResolve({ root, info, context, args }: RequestMiddlewareParams): void {
```

Comment:
Create a method for logStartOfResolve as well (like you did with logEndOfResolve)